.editorconfig

```diff
@@ -1,6 +1,10 @@
 ; Top-most EditorConfig file
 root = true
 
+[*]
+insert_final_newline = true
+end_of_line = lf
```

Comment:
All our line endings are set to platform native in git configuration.

Reply:
Ok - the issue I was having is most files were checked in LF, but there were about 6 files which were CRLF in the repo, which caused the branch to be dirty immediately on checkout. Do you have a solution for this?

Reply:
Yes, they should be in LF in the repo, so committing the LF files is the correct fix for that. Just local storage should be in native.

See https://github.com/ryzom/ryzomcore/blob/core4/.gitattributes

Reply:
Those defective files might've been committed before the .gitattributes was cleaned up.

Reply:
Amazing - I'll remove this editorconfig change then as it's not necessary